pymc_marketing/mmm/delayed_saturated_mmm.py

```diff
@@ -383,7 +383,7 @@ def channel_contributions_forward_pass(
         channel_contribution_forward_pass = (
             beta_channel_posterior_expanded * logistic_saturation_posterior
         )
-        return channel_contribution_forward_pass.eval()
+        return channel_contribution_forward_pass
```

Comment:
The type signature has to be updated. However the subclass shouldn't change the meaning of the method (by returning a numpy value).

Reply:
In the meantime, should we call the base-class method "x_untransformed", so that the sub-classes can call it without being ambiguous about the meaning?